### ISSUE TITLE
Equivalent to the Excel fill handle

### ISSUE THREAD

**Issue by ghost** [pandas-dev/pandas#22993]
#### Code Sample, a copy-pastable example if possible

```python
df.col_name.fill_down(val, start_row = 0, end_row= end)
df.loc[row_idx].fill_right(val, start_col = 0, end_col = last)
```
#### Problem description
It would be nice if there were an analog for the excel fill handle.  I am not an experienced programmer and I don't want to imply that I think the functionality should be implemented the way I suggest above, but it would be nice to have for those coming from excel to pandas.  If the val were a list, maybe it could fill repeats of the list like the fill series works in excel.
#### Expected Output
Filled values from the desired start column/row to the end or otherwise declared column/row

**Comment by svenharris** [comment]
I think that's a really interesting analogy, but I think you can get that functionality in a straightforward fashion using .iloc

**Comment by TomAugspurger** [comment]
Agreed with @svenharris.

If anyone is interested, this would make for a nice cookbook entry: http://pandas.pydata.org/pandas-docs/stable/cookbook.html

We could perhaps expand those into a "Comparison with Excel" page, or "pandas for excel users".

**Comment by mroeschke** [comment]
Agreed with @svenharris

**Comment by rotuna** [comment]
Hi, 

I would like to do this.

**Comment by simonjayhawkins** [comment]
closed in #38554
